packages/rest/src/rest.server.ts

```diff
@@ -214,7 +214,7 @@ export class RestServer extends Context implements Server, HttpServerLike {
   protected _setupRequestHandlerIfNeeded() {
     if (this._expressApp) return;
     this._expressApp = express();
-    this._expressApp.set('query parser', 'extended');
+    this._applyExpressSettings();
```

Comment:
If I am reading these lines correctly, then:

- before the change, we were explicitly configuring the Express instance to use `extended` query parser
- now we use the default `query parser` configuration as provided by Express

I see that Express uses `extended` query parser by default, so I guess this is ok.

https://expressjs.com/en/4x/api.html#app.settings.table